proposals/20230605-rules-adoption-management-maturity-framework.md

```diff
@@ -0,0 +1,222 @@
+# Falco Rules Adoption, Management and Maturity Framework
+
+## Objective
+
+The objective is to outline key enhancements and improvements to Falco, focusing on optimizing its rule adoption, customization, and management capabilities. The proposal also introduces a rules maturity framework to provide a structured approach for assessing and categorizing rules.
+
+In more detail, this proposal aims to address the expectations of adopters by providing clear guidelines for rule contribution, including processes and criteria. It also aims to provide guidance on rule customization and tuning to help adopters optimize the detection capabilities of Falco for their specific environments. Lastly, the proposal aims to empower adopters by providing them with the necessary knowledge and resources to effectively manage and customize rules. 
+
+The proposed timeline for the initial completion of each item is the Falco 0.36 release, with further improvements being continuously added based on feedback.
+
+
+## Current State
+
+This section highlights the current (as of June 6, 2023) procedures and resources for rules creation and adoption and serves as an assessment:
+
+- The Falco Project website provides detailed explanations on how to [write](https://falco.org/docs/rules/) Falco rules along with references such as the [supported fields](https://falco.org/docs/reference/rules/supported-fields/) for conditions and outputs.
+- Community members contribute rules via opening a PR against the [falcosecurity/rules](https://github.com/falcosecurity/rules) repo.
+- [Tutorials](https://falco.org/docs/tutorials/) and [blog posts](https://falco.org/tags/rules/) related to rules are available on the Falco website, along with numerous references to webinars and conference talks.
+- Falco features a CLI tool called [falcoctl](https://github.com/falcosecurity/falcoctl) to manage the lifecycle of rules (installation, updates), see [blog post](https://falco.org/blog/falcoctl-install-manage-rules-plugins/).
+- The rules framework of Falco is battle-tested and proven to be reliable in production environments. It is used by a diverse range of [organizations](https://github.com/falcosecurity/falco/blob/master/ADOPTERS.md).
+- Existing rules yaml files
+  - Main Falco rules ([falco_rules.yaml](https://github.com/falcosecurity/rules/blob/main/rules/falco_rules.yaml)) based on syscall and container events.
+  - [application_rules.yaml](https://github.com/falcosecurity/rules/blob/main/rules/application_rules.yaml) also contains rules based on network-related syscalls, which may seem misleading if you expect those rules to be present in the regular "falco_rules.yaml" file.
```

Comment:
Thanks @TheFoxAtWork -- @falcosecurity/rules-maintainers can we merge those rules into the existing falco_rules.yaml? What needs to happen to facilitate this transition?

Reply:
As discussed yesterday, `application_rules.yaml` was never meant to be part of the default ruleset (it just provides some examples) and should not be merged into `falco_rules.yaml`. I reported full details in my [review below](https://github.com/falcosecurity/rules/pull/76#pullrequestreview-1518592957) :point_down: 

A side note: as also discussed yesterday, `application_rules.yaml` should remain a distinct file and be downgraded to `sandbox` or potentially `deprecated` status. This will not cause any disruption for users, as Falco has never defaulted to loading this file, and all rules within it are commented (they were originally intended as examples and not as ready-to-use rules).

Reply:
ACK, would propose to gather some community feedback around this file and the rules it contains. In addition, looking back to the discussion @maxgio92 and I had we may wanna keep track of the "what are these application rules for Falco vs the other rules?" discussion in a dedicated issue. We should not block this proposal because of this.